b2sdk/transfer/emerge/executor.py

```diff
@@ -314,8 +314,9 @@ def _find_unfinished_file_by_plan_id(
             # FIXME: encryption is None ???
             if encryption is None or file_.encryption != encryption:
                 continue
-            if bool(legal_hold) != file_.legal_hold:
-                # when `file_.legal_hold is None` it means that `legal_hold` is unknown and we skip
+            if (legal_hold or LegalHold.UNSET) != file_.legal_hold:
```

Comment:
```
>>> (LegalHold.OFF or LegalHold.UNSET) != LegalHold.OFF
True
```

Reply:
why would we compare only enum values and not the one in the argument and the one in the file version?